src/FSLibrary/StellarDataDump.fs

```diff
@@ -55,6 +57,8 @@ type StellarFormation with
         Async.Start task
 
     member self.LaunchLogTailingTasksForPod(podName: PodName) =
+        self.sleepUntilNextRateLimitedApiCallTime ()
```

Comment:
Why do we need to sleep in the main thread here? We already sleep before reading the pod logs.

Reply:
we're make an API call `ReadNamespacedPodStatus` to the cluster here, so it needs to be rate-limited